### PR TITLE
[FE-9423] Streets and Outdoors theme under omnisci layer, but label on top

### PR DESCRIPTION
**Pull request by uyanga-gb**
# Merge Checklist
## :wrench: Issue(s) fixed:
- [ ] Author referenced issue(s) fixed by this PR:
- [ ] Fixes #https://jira.omnisci.com/browse/FE-9423

## :smoking: Smoke Test
- [ ] Works in chrome
- [ ] Works in firefox
- [ ] Works in safari
- [ ] Works in ie edge
- [ ] Works in ie 11

## :ship: Merge
- [ ] author crafted PR's title into release-worthy commit message.

### CODE REVIEW

File: src/mixins/map-mixin.js [hunk 1]
```diff
@@ -644,7 +644,15 @@ export default function mapMixin(
     const layers = _map.getStyle().layers
     for (let i = 0; i < layers.length; ++i) {
       if (layers[i].type === "symbol") {
-        firstSymbolId = layers[i].id
+        // Streets and Outdoors styles are sets of layers thus only need to make the street label layer on top of omnisci layer
+        if (
+          layers[i].id === "tunnel-oneway-arrows-blue-major" ||
+          layers[i].id === "tunnel-oneway-arrows-blue-minor"
```
Comment (CaitW) [inline src/mixins/map-mixin.js]: It feels confusing to use these two layer IDs to set the `firstSymbolId`. Is there no way to check if the map type `=== "Streets" || === "Outdoors"`?

Comment (uyanga-gb) [reply src/mixins/map-mixin.js]: Well there is a comment above to make it clear. Also, there is no information about the "Streets" and "Outdoors" within the layer in the layers array from the Mapbox. 
Screen Shot 2019-10-28 at 9 59 28 AM

Comment (CaitW) [reply src/mixins/map-mixin.js]: `_map.getStyle()` returns the name of the map:

```
{
  "version": 8,
  "name": "Mapbox Outdoors",
  "metadata": {
    "mapbox:autocomposite": true,
    "mapbox:type": "default",
    "mapbox:origin": "outdoors-v10",
    "mapbox:groups": {
      "1444934828655.3389": { "name": "Aeroways", "collapsed": true },
      "1444933322393.2852": {
        "name": "POI labels  (scalerank 1)",
        "collapsed": true
      },
```

Comment (uyanga-gb) [reply src/mixins/map-mixin.js]: @CaitW, made a change to use style name if it looks better ;)

**Pull request merged**